### ISSUE TITLE
Weapon View should list only existing properties when in view mode

### ISSUE THREAD

**Issue by Juxtaposedwords** [foundryvtt-starfinder/foundryvtt-starfinder#144]
When viewing a weapon (and not in edit mode) we should see only the viewed properties. So when viewing "Weapon Details" under "Weapon Properties" for the Advanced Autobeam Artillery you would see only the applicable terms (Which would ideally have a mouse linking to or explaining the term):

Advanced Autobeam Artillery:
**Weapon Type:** Heavy Weapon
**Weapon Properties:** Two-handed, Automatic

**Comment by wildj79** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[FEAT] Weapon View should list only existing properties when in view mode](https://trello.com/c/kVIGv6U2/38-feat-weapon-view-should-list-only-existing-properties-when-in-view-mode)

**Comment by Imper1um** [comment]
I think this is a little old, and was never closed properly.

**Comment by ian612** [comment]
You are correct, this was implemented at some point and is present as of 0.27.0
